pkg/hubble/cell/cell.go

```diff
@@ -36,6 +37,9 @@ var Cell = cell.Module(
 
 	cell.Provide(newHubbleIntegration),
 	cell.Config(defaultConfig),
+
+	// Provides Hubble file exporters.
+	exporter.Cell,
```

Comment:
I am unsure if the hubble cell or the daemon cell should provide this?

Reply:
I think it makes more sense like it is now with hubble declaring the exporters cell.